style.css

```diff
@@ -88,3 +97,310 @@ button {
+  padding: 16px 0 40px 0;
+  box-sizing: border-box;
+  align-items: center;
+  gap: 12px;
+  width: 327px;
+  height: 100%;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin-top: 88px;
+  margin-left: 16px;
+}
+
+.phase-one {
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 16px;
+  padding-left: 16px 0 16px 0;
+  align-items: center;
+}
+
+.block {
+  display: flex;
+  flex-direction: column;
+  overflow-wrap: break-word;
+}
+
+.tonic {
+  display: flex;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin-top: -0.2em;
+}
+
+.canopy {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.second-text {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  color: #344563;
+  font-size: 13px;
+  font-weight: 600;
+  padding-top: -0.4em;
+}
+
+.second-small-text {
+  display: flex;
+  color: #344563;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+  padding-top: 0;
+  width: 100%;
+}
+
+.dott {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  color: #7a869a;
+  font-size: 13px;
+  font-weight: 500;
+  line-height: 16px;
+}
+
+.second-text ul li:nth-child(2) {
+  margin-left: 30px;
+}
+
+p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.see-project-button {
+  color: #6070ff;
+  background: transparent;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  cursor: pointer;
+}
+
+.see-project-button:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.see-project-button:active {
+  background-color: #2230d2;
+}
+
+.small {
+  width: 295px;
+  height: 76px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.tags {
+  margin-bottom: 20px;
+  list-style: none;
+  padding-left: 0;
+}
+
+.tag {
+  float: left;
+  text-align: center;
+  margin-right: 20px;
+  padding-top: 5px;
+  align-items: center;
+  width: 53px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.javascript {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 8px;
+  width: 85px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.about-container {
+  display: grid;
+  grid-template-rows: auto auto;
+  background-color: #fff;
+  border-top-right-radius: 80px;
+  padding: 114px 24px;
+}
```

Comment:
- [OPTIONAL] Kindly make sure to add the `border-radius-top-right` on the background of the `About myself` section.

    - ### YOUR DESIGN
    
     ![image](https://user-images.githubusercontent.com/87197412/236644098-bf9be138-ce7a-4f9f-8af8-45e7a010dbbd.png)

    - ###FIGMA DESIGN
    
     ![image](https://user-images.githubusercontent.com/87197412/236644116-e0a5cdb9-47be-4b6c-b212-b3a0927df7ce.png)